generators/client/templates/common/README.md.jhi.client.ejs

```diff
@@ -149,6 +149,36 @@ Note: There are still a few other things remaining to do for Leaflet that we won
 
 For further instructions on how to develop with JHipster, have a look at [Using JHipster in development][].
 
+<%_ if (microfrontend) { -%>
+### Developing Microfrontend
+
+Microservices doesn't contain every required backend feature to allow microfrontends to run alone.
+You must start a pre-built gateway version or from source.
+
+Start gateway from source:
+
+```
+cd gateway
+npm run app:start # alias for ./mvnw or ./gradlew, according to the configured build-tool
+```
+
+Microfrontend's watch script is configured to watch and compile microfrontend's sources but synchronizes with gateway's frontend.
+Start it using:
+
+```
+cd microfrontend
+npm run watch
+```
+
+It's possible to run microfrontend's frontend standalone using:
+
+```
+cd microfrontend
+npm start
```

Comment:
I tried this and it doesn't seem to work. When the app starts up, I see the following in the console:

```
[HPM] Error occurred while trying to proxy request /api/account from localhost:9000 to http://localhost:8080 (ECONNREFUSED) (https://nodejs.org/api/errors.html#errors_common_system_errors)
[HPM] Error occurred while trying to proxy request /management/info from localhost:9000 to http://localhost:8080 (ECONNREFUSED) (https://nodejs.org/api/errors.html#errors_common_system_errors)
```

I think this is because the frontend is proxying to 8080 instead of 8081.

```
⠙ Generating browser application bundles (phase: setup)...[HPM] Proxy created: [
  '/api',
  '/services',
  '/management',
  '/swagger-resources',
  '/v2/api-docs',
  '/v3/api-docs',
  '/h2-console',
  '/oauth2',
  '/login',
  '/auth',
  '/health'
]  ->  http://localhost:8080
```

If I change `webpack/proxy.conf.js` to have the following, it works:

```
target: `http${tls ? 's' : ''}://localhost:8081`,
```

However, I get an error from Keycloak about an invalid redirect URI. It's passing in the following:

```
http://localhost:9000/login/oauth2/code/oidc
```

Reply:
This seems to happen because the microservice is configured to use `internal` for the client ID and that doesn't have "Standard Flow Enabled". If I enable it and add `http://localhost:*` as a redirect URI, it shows me a login form and I can login. However, after I'm redirect back to the app, it doesn't show me as logged in.

Reply:
Tested with jwt, need to adjust for oauth than.